tests/SubjectFilterTest.php

```
@@ -1,314 +1,397 @@
     }
 
-    public function test__invoke()
+    public function test__invoke(): void
     {
         $this->filter->validate('foo')->is('alnum')->asSoftRule();
         $this->filter->validate('foo')->is('strlenMin', 6)->asHardRule();
 
         // check for success
-        $subject = (object) array('foo' => 'foobar');
+        $subject = (object) ['foo' => 'foobar'];
         $result = $this->filter->__invoke($subject);
```

PHPStan (2.1.54)

[error] Result of method Aura\Filter\SubjectFilter::__invoke() (void) is used.


(method.void)